roles/bibdata/molecule/default/verify.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+  - name: check bibdata software status
+    package:
+      name: "{{ item }}"
+      state: present
+    check_mode: true
+    register: pkg_status
+    loop:
+      - cifs-utils
```

Comment:
Any reason you did not test 
```
    "libmysqlclient-dev",
    "python-mysqldb",
```

Reply:
we use postgresql now @carolyncole

Reply:
and bizarrely I don't see the postgresql drivers. I will update this with those and remove the mysqldb